src/app/banners/BannerWrapper.tsx

```diff
@@ -2,15 +2,19 @@ import { useEffect, useState } from 'react';
 import Banner from './Banner';
 
 const BannerWrapper = () => {
-  const [showBanner, setShowBanner] = useState(true);
+  const [showBanner, setShowBanner] = useState(false);
 
   const onClose = () => {
     setShowBanner(false);
     localStorage.setItem('showLifetimeBanner', 'false');
   };
 
   useEffect(() => {
-    if (localStorage.getItem('showLifetimeBanner') === 'false') {
+    if (!localStorage.getItem('showLifetimeBanner')) {
+      setTimeout(() => {
+        setShowBanner(true);
+      }, 30000);
+    } else if (localStorage.getItem('showLifetimeBanner') === 'false') {
       setShowBanner(false);
     }
```

Comment:
Now that it is closed by default, is it necessary to check if the value of the local storage is false when the component is mounted? I think this logic is no longer necessary